### DISCUSSION TITLE
issue while installing scripts on server

### DISCUSSION THREAD

**Discussion by Maheshdvp999** [learndevopsonline/learndevopsonline#1330] [Q&A] [answered]
Hi team I had updated my script with ip addresses and wand to install that in each server but facing this issue could you please help me that where i did my mistake.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/134056076/fce1fc18-0f62-44bd-9ecf-a118daa97d6f)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/134056076/50c4783d-0efa-459c-853f-e22195638f39)

**Comment by maheshbusireddy** [comment, marked as answer]
Hi @Maheshdvp999 Error in your script 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/99891108/e29b4ad1-6cd6-4b25-ae85-da4c74fbd30b)

### cp mysql.repo /etc/yum.repos.d/mysql.repo
you have to use mongo repo file but you used mysql repo file.
